chirp/drivers/kg935g.py

```diff
@@ -430,6 +412,35 @@
 # MRT 1.2 correct spelling of Wouxon
 
 
+def str2callid(val):
+    """ Convert caller id strings from callid2str.
+    """
+    ascii2bin = "0123456789"
+    s = str(val).strip()
+    LOG.debug("val = %s" % val)
+    LOG.debug("s = %s" % s)
+    if len(s) < 3 or len(s) > 6:
+        raise InvalidValueError(
+            "Caller ID must be at least 3 and no more than 6 digits")
+    if s[0] == '0':
+        raise InvalidValueError(
+            "First digit of a Caller ID cannot be a zero '0'")
+    blk = bytearray()
+    for c in s:
+        if c not in ascii2bin:
+            raise InvalidValueError(
+                "Caller ID must be all digits 0x%x" % c)
+        b = (0x0, 0x1, 0x2, 0x3, 0x4, 0x5, 0x6, 0x7, 0x8, 0x9)[int(c)]
```

Comment:
This is what you want here:
```
b = asc2bin.index(c)
```

Reply:
will fix right now

Reply:
where does asc2bin come from.. not seeing it code
do I need to import it?

Reply:
Oh sorry I meant `ascii2bin`, L418